### PR TITLE
Aos fixups mmc driver

### PR DESCRIPTION
**Pull request by xakep-amatop**

### CODE REVIEW

**Comment by xakep-amatop** [comment]
prev pull is here https://github.com/xen-troops/zephyr/pull/38
note: add a few extra commits

File: include/zephyr/drivers/gpio.h [hunk 1]
```diff
@@ -428,6 +428,7 @@ struct gpio_dt_spec {
  * in the device structure.
  */
 struct gpio_driver_config {
+	DEVICE_MMIO_ROM; /* Must be first */
```
Comment (lorc) [inline include/zephyr/drivers/gpio.h:431]: This should be in a separate commit with big fat explanation why this is the correct change.

File: include/zephyr/drivers/gpio.h [hunk 2]
```diff
@@ -441,6 +442,7 @@ struct gpio_driver_config {
  * element in the driver's struct driver_data declaration.
  */
 struct gpio_driver_data {
+	DEVICE_MMIO_RAM; /* Must be first */
```
Comment (lorc) [inline include/zephyr/drivers/gpio.h:445]: The same also should go into the same separate commit

**Review by lorc — COMMENTED** [review]
I'm a bit worried about large changes in fixups. Are you sure that original commit descriptions will be still correct after fixups applied?

**Comment by xakep-amatop** [comment]
all fixup commits have been already moved to pull https://github.com/xen-troops/zephyr/pull/43 
R-car gpio driver related changes you can find here https://github.com/xen-troops/zephyr/pull/34
so, I am closing this one
